### PR TITLE
Allowing to Create multiple Spaniel Instances

### PR DESCRIPTION
**Pull request by SparshithNR**
Please visit the [RFC](https://github.com/SparshithNR/spaniel/blob/performance/RFC-performance.md) for this change and share your view.

### CODE REVIEW

File: src/intersection-observer.ts [hunk 1]
```diff
@@ -67,7 +71,7 @@ export class SpanielIntersectionObserver implements IntersectionObserver {
   private id: string;
   private scheduler: ElementScheduler;
   private callback: Function;
-
+  public parent: SpanielInstance;
```
Comment (asakusuma) [inline src/intersection-observer.ts]: Does this need to be public?

Comment (SparshithNR) [inline src/intersection-observer.ts]: No, It must be private. I will make the change. Even in spaniel-observer I have to make the same change

**Comment by SparshithNR** [comment]
Please visit the RFC for this change and share your view.
https://github.com/SparshithNR/spaniel/blob/performance/RFC-performance.md

**Comment by asakusuma** [comment]
Moving to another PR https://github.com/linkedin/spaniel/pull/63

**Pull request closed without merging**